utilities/test_suite/HIP/runTests.py

```diff
@@ -35,6 +36,53 @@
 inFilePath2 = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src2')
 qaInputFile = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src1')
 
+import os
+import subprocess  # nosec
+import argparse
+import sys
+import datetime
+import shutil
+
+# Set the timestamp
+timestamp = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+
+# # Set the value of an environment variable
+# os.environ["TIMESTAMP"] = timestamp
+
+cwd = os.getcwd()
+inFilePath1 = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src1')
+inFilePath2 = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src2')
+qaInputFile = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src1')
+
+def validate_and_remove_contents(path):
+    if not path:  # check if a string is empty
+        print("Folder path is empty.")
+        exit()
+    if path == "/*":  # check if the root directory is passed to the function
+        print("Root folder cannot be deleted.")
+        exit()
+    if os.path.exists(path):  # check if the folder exists
+        os.system("rm -rvf {}/*".format(path))  # Delete the directory if it exists
+    else:
+        print("Path is invalid or does not exist.")
+        exit()
+
+def validate_and_remove_folders(path, folder):
+    if path == "/*":  # check if the root directory is passed to the function
+        print("Root folder cannot be deleted.")
+        exit()
+    if path and os.path.isdir(path + "/.."):  # checks if directory string is not empty and it exists
+        output_folders = [folder_name for folder_name in os.listdir(path + "/..") if folder_name.startswith(folder)]
+
+        # Loop through each directory and delete it only if it exists
+        for folder_name in output_folders:
+            folder_path = os.path.join(path, "..", folder_name)
+            if os.path.isdir(folder_path):
+                os.system("rm -rf {}".format(folder_path))  # Delete the directory if it exists
+                print("Deleted directory:", folder_path)
+            else:
+                print("Directory not found:", folder_path)
+
 def case_file_check(CASE_FILE_PATH):
```

Comment:
Lets add one liner comments above each of these def functions stating what they do

Reply:
Done